README.rst

```diff
@@ -1,817 +1,54 @@
-Create a new ``IP Address``.
-
-.. code-block:: text
-
-    /api/v1/ipam/subnet/<subnet_id>/ip-address/
-
-===========    ========================================
-Param          Description
-===========    ========================================
-ip_address     IPv6/IPv4 address value
-subnet         Subnet UUID
-description    Optional description for the IP address
-===========    ========================================
-
-Subnet List/Create View
-=======================
-
-An api endpoint to create or retrieve the list of subnet instances.
-
-GET
----
-
-Returns the list of ``Subnet`` instances.
-
-.. code-block:: text
-
-    /api/v1/ipam/subnet/
-
-POST
-----
-
-Create a new ``Subnet``.
-
-.. code-block:: text
-
-    /api/v1/ipam/subnet/
-
-=============    ========================================
-Param            Description
-=============    ========================================
-subnet           Subnet value in CIDR format
-master_subnet    Master Subnet UUID
-description      Optional description for the IP address
-=============    ========================================
-
-Subnet View
-===========
-
-An api endpoint for retrieving, updating or deleting a subnet instance.
-
-GET
----
-
-Get details of a ``Subnet`` instance
-
-.. code-block:: text
-
-    /api/v1/ipam/subnet/<subnet-id>/
-
-DELETE
-------
-
-Delete a ``Subnet`` instance
-
-.. code-block:: text
-
-    /api/v1/ipam/subnet/<subnet-id>/
-
-PUT
----
-
-Update details of a ``Subnet`` instance.
-
-.. code-block:: text
-
-    /api/v1/ipam/subnet/<subnet-id>/
-
-=============    ========================================
-Param            Description
-=============    ========================================
-subnet           Subnet value in CIDR format
-master_subnet    Master Subnet UUID
-description      Optional description for the IP address
-=============    ========================================
-
-IP Address View
-===============
-
-An api endpoint for retrieving, updating or deleting a IP address instance.
-
-GET
----
-
-Get details of an ``IP address`` instance.
-
-.. code-block:: text
-
-    /api/v1/ipam/ip-address/<ip_address-id>/
-
-DELETE
-------
-
-Delete an ``IP address`` instance.
-
-.. code-block:: text
-
-    /api/v1/ipam/ip-address/<ip_address-id>/
-
-PUT
----
-
-Update details of an ``IP address`` instance.
-
-.. code-block:: text
-
-    /api/v1/ipam/ip-address/<ip_address-id>/
-
-===========    ========================================
-Param          Description
-===========    ========================================
-ip_address     IPv6/IPv4 value
-subnet         Subnet UUID
-description    Optional description for the IP address
-===========    ========================================
-
-Export Subnet View
-==================
-
-View to export subnet data.
+    :alt: Feature Highlights
 
-POST
-----
+Refer to `IPAM: Features
+<https://openwisp.io/docs/__new__/dev/ipam/user/intro.html>` of OpenWISP
+documentation for a complete overview of features.
```

Comment:
This sentence doesn't sound right to me, have you run it through AI tools to make it better?

Remember: the structure of the READMEs need to be similar, I think we are missing the links to the usage docs and developer docs here.